README.md

```
@@ -11,262 +6,60 @@ Convert your Cursor rules to Claude Code config, or vice versa. Take your agent
-|------|--------|--------|
-| Cursor | `@a16njs/plugin-cursor` | ✅ Bundled |
-| Claude Code | `@a16njs/plugin-claude` | ✅ Bundled |
-| Codex | Community | 🔌 [See plugins](#community-plugins) |
-| Windsurf | Community | 🔌 [See plugins](#community-plugins) |
-
-## Understanding Conversions
-
-Different tools have different capabilities. a16n handles this transparently.
+| Tool | Status |
+|------|--------|
+| Cursor | ✅ Supported |
```

markdownlint-cli2 (0.20.0)

[warning] Table column style
Table pipe does not align with header for style "aligned"

(MD060, table-column-style)